packages/gatsby/static/configuration/manifest.json

```diff
@@ -53,113 +55,98 @@
       "description": "A field used to expose some executable Javascript files to the parent package. Any entry listed here will be made available through the `$PATH`. Note that it is very advised to only expose Javascript files for portability reasons (shellscripts and non-js binaries require the user to have a specific shell, and are incompatible with zip access).",
       "type": "object",
       "patternProperties": {
-        "^(?:(.+))$": {
+        "^(.+)$": {
           "type": "string",
           "format": "uri-reference",
-          "default": "./dist/my-bin.js"
+          "examples": ["./dist/my-bin.js"]
         }
       },
       "additionalProperties": false,
-      "default": {
-        "my-bin": {}
-      },
-      "$comment": "{ \"margin\": false }"
+      "exampleKeys": ["my-bin"],
```

Comment:
Just to be sure, `exampleKeys` is an extension over JSON schema, right? It'll just be ignored by interpreters other than our own?

Reply:
Yep, `exampleKeys` is an extension. I've prefixed all extension properties with an underscore to remove the confusion and also added an `__info` property to both schemas to explain a few things.